src/main/app/src/constants/ActivitiesFilterBar.js

```diff
@@ -20,11 +21,12 @@ const ActivitiesFilterBar = ({activityType, activityTypes, links, updateFiltered
         linkFilters.numOfPLayers = numOfPLayers;
         break;
       case activityTypes.VIDEOS:
-        linkFilters.videoType = videoType;
+        linkFilters.videoCategory = videoCategory;
```

Comment:
Maybe add a comment listing the video categories we are using? "// Yoga, Workout, Meditation"

Reply:
Done!